hack/prebuild/init.sh

```diff
@@ -0,0 +1,19 @@
+#/bin/bash
+
+yum install -y -d1 libvirt libvirt-daemon-kvm
+usermod -aG libvirt root
+
+cat <<EOF > /etc/libvirt/libvirtd.conf
+unix_sock_group = "libvirt"
+unix_sock_rw_perms = "0770"
+listen_tls = 0
+listen_tcp = 1
+auth_tcp="none"
+tcp_port = "16509"
+EOF
+
+echo 'LIBVIRTD_ARGS="--listen"' >> /etc/sysconfig/libvirtd
+
+iptables -I INPUT -p tcp --dport 16509 -j ACCEPT -m comment --comment "Allow insecure libvirt clients"
+
+systemctl start libvirtd
```

Comment:
Do we also need `systemctl enable libvirtd`? Is there any expectation of this starting on reboot?

Reply:
No, those machines aren't expected to be rebooted.